data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/leader/LeaderScheduler.java

```diff
@@ -116,21 +115,6 @@ public void run() {
 
     public void shutdown() {
         shutdownRequested = true;
-
-        // Clean up publication and replication slot for Postgres
-        if (streamPartition != null) {
```

Comment:
How do you plan to clean up long term when the RDS source is deleted ?

Reply:
I don't have a good plan on this yet. We can discuss this offline.